pdf_build_src/process_markdowns.py

```diff
@@ -141,25 +142,34 @@ def modify_changelog():
         file.writelines(data)
 
 
-"""Number of chars maximal in one line approximated from a line of the PDF"""
+# Number of chars maximal in one line approximated from a line of the PDF
 NB_CHARS_LINE_PDF = 100
 
-def correct_table(table):
-    """Compute the max number of characters and dashes for each column and create the corrected table
+def correct_table(table, offset = [10, 40], debug=False):
+    """Create the corrected table.
+    It computes the number of characters maximal in each column and reformat line to make sure 
```

Comment:
```suggestion
    """Create the corrected table.
    
    It computes the number of characters maximal in each column and reformat line to make sure 
```

you need a blank line between the first line of the docstring and the next part of the docstring. This is important for the behavior when you want "help" about a function. It'll first display the first line only, then the rest --> and that's what we want.

Else it'll display everything at once in weird formatting.